modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/ScheduleModal.js

```diff
@@ -8,16 +8,49 @@ import ClayButton from '@clayui/button';
 import ClayDatePicker from '@clayui/date-picker';
 import ClayModal from '@clayui/modal';
 import classnames from 'classnames';
-import {isAfter} from 'date-fns';
 import PropTypes from 'prop-types';
 import React, {useEffect, useState} from 'react';
 
+const DIFFERENCE_IN_YEARS = 1;
+
+export function isValidDate(dateString) {
+
+	// Regular expression for the 'yyyy-MM-dd HH:mm' format
+
+	const dateRegex = /^(\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1]) (0[0-9]|1[0-9]|2[0-3]):([0-5][0-9])$/;
+
+	if (!dateRegex.test(dateString.trim())) {
+		return false;
+	}
+
+	const [, year, month, day, hour, minute] = dateString
```

Comment:
Is this a typo? `const [, year, ...`

Reply:
Nop, I'm ignoring on propose the first two characters of the date `(19|20)` because the Date constructor expects the year in `0` to `99` format, you can get more info at [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#year)